WTWSMS/history/characters/roman.txt

```diff
@@ -106,14 +106,14 @@
 	name="Anicius" # PLRE vol. 2 Anicius Manlius Severinus Boethius iunior 5
 	# Major Poet, usually called Boethius
 	# Consul of 510
-	#dynasty=1042312
+	dynasty=1042285
 	religion="nicene"
 	culture="roman"
 	father=20783
 	479.1.1={
 		birth="479.1.1"
 	}
-	500.1.1 = {
+	490.1.1 = {
```

Comment:
It was edited because their children are born in 490, right?

Reply:
Yes

Reply:
Maybe the children should just get different birth dates instead of this? A man getting married and having children in the age of 11 seems strange to me, even in Roman times.

Reply:
Yes, they were reported as a children in time of their consulship. But i think might be better make 500 as a their birth date.